### PR TITLE
SPCTR-2185 - New: InfoBox 2.0 block stylings

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
New: Info Box 2.0 block

### Types of changes
 New feature (non-breaking change which adds functionality) -->

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: includes/blocks/new-block/attributes.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Attributes File.
+ *
+ * @since 2.0.0
```
Comment (AbhijitNage123) [inline includes/blocks/new-block/attributes.php:5]: Since x.x.x

File: includes/blocks/new-block/block.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Block Information & Attributes File.
+ *
+ * @since 2.0.0
```
Comment (AbhijitNage123) [inline includes/blocks/new-block/block.php:5]: since x.x.x

File: includes/blocks/new-block/frontend.css.php [hunk 3]
```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * Frontend CSS & Google Fonts loading File.
+ *
+ * @since 2.2
```
Comment (AbhijitNage123) [inline includes/blocks/new-block/frontend.css.php:5]: since x.x.x

File: includes/blocks/new-block/block.php (hunk 2 above)
Comment (supriyajjadhav) [reply includes/blocks/new-block/block.php]: updated

**Review by AbhijitNage123 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**